src/Components/SingleArticle.jsx

```diff
@@ -12,15 +13,27 @@ function SingleArticle() {
         getArticleByID(article_id)
         .then((response) => {
             setArticle(response.data.article[0])
-        }, [article_id])
-    })
+            // console.log(response.data.article[0])
```

Comment:
It would be great to have a loading state here to show users that the request is ongoing until it completes 💡